packages/core/integration-tests/test/integration/fs-destructure/index.js

```diff
@@ -1,2 +1,2 @@
-var {readFileSync} = require('fs');
+var readFileSync = require('fs').readFileSync;
```

Comment:
Other changes look fine, but this file was correct. If the test fails, then because Parcel isn't working correctly.

Reply:
Figured that. I wanted to see if the CI is gonna pass when I fix every failed test I found on my local machine. Turns out there's a lint error on Windows but I don't have the OS to debug that 😓 

That aside, I didn't know where to look at in order to fix this. Can you give me a hint? 🙏